docs/Recipes.md

```diff
@@ -142,6 +145,35 @@ export default makeConfig({
 });
 ```
 
+### With React Native 0.60 and newer
+
+React Native 0.60 removed Haste imports, so now in order to use out-of-tree platforms, you need to modify `InitializeCore` location and alias `react-native` to your out-of-tree library.
```

Comment:
Technically they removed _some_ of the imports, all of them were removed in 0.61. Not sure if it makes sense to make it public :D